### PR TITLE
Homework 2 arrays

### PR DESCRIPTION
**Pull request by Denis-Pikus**
My homework ArrayService

### CODE REVIEW

File: IntensiveJavaGuru/src/main/java/dpdev/homework_2/ArrayServiceDemo.java [hunk 1]
```diff
@@ -0,0 +1,17 @@
...
+
+
+public class ArrayServiceDemo {
+    public static void main(String[] args) {
+//        ArrayService arrayService = new ArrayService();
+//        int [] arr = arrayService.create(10);
+//        arrayService.fillRandomly(arr);
+//        System.out.println(Arrays.toString(arr));
+//        arrayService.swap(arr);
+//        System.out.println(Arrays.toString(arr));
```
Comment (VladimirPetranovski) [inline IntensiveJavaGuru/src/main/java/dpdev/homework_2/ArrayServiceDemo.java:15]: no comment

File: IntensiveJavaGuru/src/main/java/dpdev/homework_2/service/ArrayService.java [hunk 2]
```diff
@@ -0,0 +1,64 @@
+package dpdev.homework_2.service;
+
+import java.util.Arrays;
+import java.util.Random;
+
+public class ArrayService {
+
+    public int[] create(int size) {
+        if (size < 0 ) {
```
Comment (VladimirPetranovski) [inline IntensiveJavaGuru/src/main/java/dpdev/homework_2/service/ArrayService.java:9]: <= 0

File: IntensiveJavaGuru/src/main/java/dpdev/homework_2/service/ArrayService.java [hunk 3]
```diff
@@ -0,0 +1,64 @@
...
+            throw new NegativeArraySizeException("It impossible to create array which negative size!");
+        } else return new int[size];
+    }
+
+    public void fillRandomly(int[] array) {
+        //array = new Random().ints(array.length, 0, 101).toArray();
```
Comment (VladimirPetranovski) [inline IntensiveJavaGuru/src/main/java/dpdev/homework_2/service/ArrayService.java:15]: no comment

**Review by VladimirPetranovski — APPROVED** [review]
lgfm

File: IntensiveJavaGuru/pom.xml [hunk 4]
```diff
@@ -9,20 +9,21 @@
     <version>1.0-SNAPSHOT</version>
 
     <properties>
-        <sonar.projectKey>javaGuruBY_java-free-stream-5</sonar.projectKey>
-        <sonar.organization>javaguruby</sonar.organization>
+        <sonar.projectKey>Denis-Pikus_JavaGuruIntensive2021</sonar.projectKey>
```
Comment (integratorBY) [inline IntensiveJavaGuru/pom.xml:12]: 👍

**Review by integratorBY — APPROVED** [review]
lgfm

**Pull request merged**